tests/queries/0_stateless/03305_rename_mutations_counter_replicated.reference

```diff
@@ -0,0 +1,3 @@
+ActiveMetadataMutations	1
+ActiveMetadataMutations	0
```

Comment:
maybe we can also add a test for ActiveDataMutations?

Reply:
Tests for `ActiveDataMutations` are added in private repo.

Reply:
I'll move it to public after #74877 is merged.